mkdocs/docs/configuration.md

```diff
@@ -84,6 +84,7 @@ For the FileIO there are several configuration options available:
 | s3.session-token     | AQoDYXdzEJr...           | Configure the static session token used to access the FileIO.                                                                                                                                                                                             |
 | s3.signer            | bearer                   | Configure the signature version of the FileIO.                                                                                                                                                                                                            |
 | s3.signer.uri        | http://my.signer:8080/s3 | Configure the remote signing uri if it differs from the catalog uri. Remote signing is only implemented for `FsspecFileIO`. The final request is sent to `<s3.singer.uri>/v1/aws/s3/sign`.                                                                |
+| s3.signer.endpoint   | v1/main/s3-sign          | Configure the remote signing endpoint. Remote signing is only implemented for `FsspecFileIO`. The final request is sent to `<s3.singer.uri>/<s3.singer.endpoint>`. Default to v1/aws/s3/sign`.                                                             |
```

Comment:
what happens if im using pyarrow for fileio?

Reply:
also s/s3.singer.endpoint/s3.signer.endpoint

Reply:
> what happens if im using pyarrow for fileio?

The property will be ignored

Reply:
fix it.